src/AppBundle/Entity/Answer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: lukas
+ * Date: 16.11.13
+ * Time: 22.30
+ */
+
+namespace AppBundle\Entity;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="answers")
+ */
+class Answer
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\ManyToOne(targetEntity="questions")
+     */
```

Comment:
On relation type fields, you don't need to provide Column definition. Also targetEntity must be Class name of mapped target, not db table name. Also ManyToOne also needs an inversedBy property, witch defines back relationship http://symfony.com/doc/current/doctrine/associations.html